src/Projection/DefaultProjectionHandler.php

```diff
@@ -0,0 +1,130 @@
+    {
+        foreach ($this->projections as $projection) {
+            if ($onlyProjections && !in_array($projection::class, $onlyProjections)) {
+                continue;
+            }
+
+            $metadata = $this->metadata($projection);
+
+            if (!array_key_exists($event::class, $metadata->handleMethods)) {
+                continue;
+            }
+
+            $method = $metadata->handleMethods[$event::class];
+
+            $projection->$method($event);
+        }
+    }
+
+    public function create(?array $onlyProjections = null): void
+    {
+        foreach ($this->projections as $projection) {
+            if ($onlyProjections && !in_array($projection::class, $onlyProjections)) {
+                continue;
+            }
+
+            $metadata = $this->metadata($projection);
+            $method = $metadata->createMethod;
+
+            if (!$method) {
+                continue;
+            }
+
+            $projection->$method();
+        }
+    }
+
+    public function drop(?array $onlyProjections = null): void
+    {
+        foreach ($this->projections as $projection) {
+            if ($onlyProjections && !in_array($projection::class, $onlyProjections)) {
+                continue;
+            }
+
+            $metadata = $this->metadata($projection);
+            $method = $metadata->dropMethod;
+
+            if (!$method) {
+                continue;
+            }
+
+            $projection->$method();
+        }
+    }
+
+    public function metadata(Projection $projection): ProjectionMetadata
+    {
+        if (array_key_exists($projection::class, $this->projectionMetadata)) {
+            return $this->projectionMetadata[$projection::class];
+        }
+
+        $reflector = new ReflectionClass($projection::class);
+        $methods = $reflector->getMethods();
+
+        $metadata = new ProjectionMetadata();
+
+        foreach ($methods as $method) {
+            $attributes = $method->getAttributes(Handle::class);
+
+            foreach ($attributes as $attribute) {
+                $instance = $attribute->newInstance();
+                $eventClass = $instance->aggregateChangedClass();
+
+                if (array_key_exists($eventClass, $metadata->handleMethods)) {
+                    throw new DuplicateHandleMethod(
+                        $projection::class,
+                        $eventClass,
+                        $metadata->handleMethods[$eventClass],
+                        $method->getName()
+                    );
+                }
+
+                $metadata->handleMethods[$eventClass] = $method->getName();
+            }
+
+            if ($method->getAttributes(Create::class)) {
+                $metadata->createMethod = $method->getName();
+            }
+
+            if (!$method->getAttributes(Drop::class)) {
+                continue;
+            }
+
+            $metadata->dropMethod = $method->getName();
```

Comment:
If adding the Attribute to more then 1 Method in the class then the last one would be taken, i would suggest to throw an exception here, since it makes literally no sense to provide multiple create or drop methods. This only would lead to wtf moments so we should be here really explicit imho.